### ISSUE TITLE
NVDA 64-bit does not recognize 32-bit synthesizers - including all SAPI 4

### ISSUE THREAD

**Issue by titet11** [nvaccess/nvda#19030]
### Is your feature request related to a problem? Please describe.

Yes, no 32-bit speech synthesizers work on 64-bit versions of NVDA.
This includes all SAPI 4 synths

Version used:
nvda_snapshot_alpha-52937,f535af1f.exe61.9

What speech synthesizers did you use to detect the issue?

I used "Eloquence TTS" for SAPI 5 and the "Eloquence" plugin created by @davidacm and pumper42 nikel.

Question:

I understood that they would offer support for 32-bit synthesizers, but there is no note about it.

What should be done in this case? Will any support be implemented?

### Describe the solution you'd like

32-bit synthesizers should work on 64-bit versions of NVDA.

### Describe alternatives you've considered

I think any alternative would increase the latency of 32-bit synthesizers, and would be quite complex to implement, So I think if you guys support this issue, everything could get back to normal with the lowest possible latency.

**Comment by CyrilleB79** [comment]
@SaschaCowley, @seanbudd, this issue is closed as completed. It has not been completed at all since it remain impossible to use 32-bit synths.

Either it should remain open (best solution given the expectation of people), or it should be closed as "won't fix", given the fix is not expected in NVDA but in an add-on.

Since the add-on is expected from NV Access (Mick), I'd prefer keeping this issue open so that the progress of this solution can be tracked.

**Comment by pitermach** [comment]
I am honestly a little confused and disappointed about what just happened. Up to this point, the way I've understood it was that NVAccess was taking this issue really seriously, especially because it was marked as blocking. So I assumed that no beta would come out until this compatibility add-on was ready, which I thought was a really good thing. 

Many people still rely on older speech synthesizers for various reasons. There are languages where the only good speech options are older legacy synthesizers, and many people who use  older voices due to a hearing impairment, because they are more  intelligible. For these people using Espeak, OneCore or purchasing a commercial speech addon may simply not be an option. The situation is even more serious for older braille displays due to how much they cost, making those previously mentioned people even more affected. So seeing this issue being completed by unceremoniously removing the drivers makes me a bit concerned. Hopefully I’m misunderstanding the situation and the plan is still to have a compatibility addon, and if that’s the case I agree with @CyrilleB79 that this issue should remain open until this is done. If you don't want to delay the release for this and the issue was closed just to unblock, then I think the documentation should be updated with a more clear indication that although support was removed it will be reintroduced in the near future as an addon.

**Comment by CyrilleB79** [comment]
OK @michaelDCurran. Many thanks for the feedback and for the hard work!

**Comment by tmthywynn8** [comment]
If 32-bit is no longer being supported from NVDA 2026.1 onwards, what does this mean for SAPI5 voices that don't have a 64-bit build? I'm thinking specifically, of, say, [Sao Mai's VNVoice](https://saomaicenter.org/en/smsoft/vnvoice) which "is the first usable text-to-speech engine with screen reading software on Windows for Vietnamese language." And according to the same page, "it's still being used widely by blind computer users in Vietnam till today!"
